scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -53,7 +53,7 @@ workflow GvsJointVariantCalling {
             input_vcf_indexes = input_vcf_indexes,
             input_vcfs = input_vcfs,
             filter_set_name = filter_set_name,
-            use_classic_VQSR = true,
+            use_classic_VQSR = use_classic_VQSR,
```

Comment:
Nothing to change for this PR, but over in my forthcoming PR I propose to change the name of this variable internally to [use_VQSR_lite](https://github.com/broadinstitute/gatk/blob/b95a80dd5745b2781a28f9275a95fd6d83dc1878/scripts/variantstore/wdl/GvsJointVariantCalling.wdl#L56). Would we want to maintain the `use_classic_VQSR` name at this level or roll with `use_VQSR_lite` or something else?